### ISSUE TITLE
How many step does it take to converge?

### ISSUE THREAD

**Issue by YuZhang10** [google-research/motion_imitation#7]
Hi, I have trained the policy for over 50M steps, but it seems not converge yet.

I wonder how many steps in total it need to finish trianing?

![image](https://user-images.githubusercontent.com/30391848/83343964-b09d9b80-a333-11ea-9273-39e81efcf94d.png)
(BTW, I use 16 way parallel training)

Thanks.

**Comment by erwincoumans** [comment]
Somewhere around 100 million samples and for some of the more complex motions even more (150-200 million). Note that quality of the policies is not as good as our internal version we used for the paper, it may require a bit more hyper-parameter tuning.

**Comment by YuZhang10** [comment]
Thanks for your reply.

I've noticed that the trained lakaigo can already walk smoothly while the vf loss constantly increases. Does it make any sense? or it means I did something wrong?

**Comment by TimeEscaper** [comment]
Dear @erwincoumans,

Can you please clarify the parameters of your computational setup that you used to training in original paper? And how much time did training take for one motion skill?

**Comment by erwincoumans** [comment]
Computation was in the order of day(s) typically. The computational setup are a cluster cloud machines, similar to Google Cloud.
